Source/CesiumRuntime/Public/Cesium3DTileset.h

```diff
@@ -1127,6 +1128,14 @@ class CESIUMRUNTIME_API ACesium3DTileset : public AActor {
   void UpdateTransformFromCesium();
 
 private:
+  /**
+   * The event handler for ACesiumGeoreference::OnEllipsoidChanged.
+   */
+  UFUNCTION()
+  void _onGeoreferenceEllipsoidChanged(
```

Comment:
The naming on this function looks stylistically inconsistent.

It is a handler for an event named `OnEllipsoidChanged`. For reference, one existing example of a callback that adds itself to the `OnGeoreferenceUpdated` event is `HandleGeoreferenceUpdated`.

Maybe `HandleEllipsoidChanged` would be better?

Reply:
It's named this way to match `UCesiumGlobeAnchorComponent::_onGeoreferenceChanged` and `_onActorChanged`. But neither of these match Unreal's naming conventions, so I'm fine changing it.

Reply:
Thanks. I _think_ this is more consistent now?